src/Aqua.jl

```diff
@@ -77,33 +65,31 @@ passed to `\$x` to specify the keyword arguments for `test_\$x`.
 - `ambiguities = true`
 - `unbound_args = true`
 - `undefined_exports = true`
-- `piracy = $(_lt06("false", "true"))`
-- `project_extras = $(_lt05("false", "true"))`
-- `stale_deps = $(_lt05("false", "true"))`
-- `deps_compat = $(_lt05("false", "true"))`
-- `project_toml_formatting = $(_lt05("false", "true"))`
+- `piracy = true`
```

Comment:
Could you update the order here s.t. it is equal to the execution order. E.g. piracy should be the last.

Reply:
Done